json-schema/Request-Context-v2.json

```diff
@@ -0,0 +1,129 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "https://github.com/tewhatuora/schemas/json-schema/Request-Context",
+    "description": "JSON schema for request context custom header schema for apps integrating with HNZ FHIR APIs",
+    "type": "object",
+    "properties": {
+      "userIdentifier": {
+        "type": "string",
+        "description": "The user Id as known to the system they have signed into (which is consuming the FHIR API)",
+        "example": "janedoe@bigmedicalcentre.co.nz"
+      },
+      "userRole": {
+        "type": "string",
+        "enum": [
+            "PAT",
+            "PROV"
+        ]
+      },
+      "hpiPractitioner": {
+        "type": "object",
+        "description": "The user's HPI health practitioner identification",
+        "properties": {
+          "cpnNumber": {
+            "type": "string",
+            "description": "The HPI Common Person Number (aka HPI Practitioner Index) in the form NNXXXX where N is numeric and X is Alphabetic",
+            "example": "90ZZLP"
+          },
+          "registrationAuthorityNumber": {
+            "type": "string",
+            "description": "The number issued to the practitioner by their registration authority",
+            "example": "123456"
+          },
+          "registrationAuthority": {
+            "type": "object",
+            "description": "Details of the registration authority aka registration council",
+            "properties": {
+              "raUrl": {
+                "type": "string",
+                "description": "The system Url for the practitioner's registration authority/council",
+                "example": "https://standards.digital.health.nz/ns/medical-council-id"
+              }
+            }
+          }
+        },
+        "required": [ "cpnNUmber", "registrationAuthorityNumber", "registrationAuthority" ]        
+      },
+      "hpiFacility": {
```

Comment:
Might need to add hpiOrganisation as well

Reply:
yeah I thought that might come up - may as well get that in now too